ui/widgets/nuxeo-tooltip.js

```diff
@@ -132,10 +135,15 @@ import { microTask } from '@polymer/polymer/lib/utils/async.js';
     }
 
     hide() {
-      if (this._tooltip) {
-        document.body.removeChild(this._tooltip);
-        this._tooltip = null;
+      const paperToolTip = document.getElementsByTagName('paper-tooltip');
```

Comment:
Kindly dont change the old code instead add below code above old code

     if (this._tooltip == undefined && document.getElementsByTagName('paper-tooltip') && document.getElementsByTagName('paper-tooltip')[0]) {
        this._tooltip =document.getElementsByTagName('paper-tooltip')[0];
     }